additionalTypeDefs/ResourceMarkdown.graphqls

```diff
@@ -0,0 +1,13 @@
+extend type ResourceMarkdown {
+    """
+    Resolved MediaRecords referenced in the ResourceMarkdown text in order.
+    A MediaRecord might be NULL if the referenced MediaRecord no longer exists. 
+    """
+    referencedMediaRecords: [MediaRecord]! @resolveTo(
+        sourceName: "MediaService",
+        sourceTypeName: "Query",
+        sourceFieldName: "mediaRecordsByIds",
```

Comment:
Does this still exist? Didn't you rename it to "findMediaRecordsByIds"?

Reply:
Both queries still exist but you are correct in that this resolver should use the find... query because media records can be non-existant